pandas/core/tools/numeric.py

```diff
@@ -188,6 +197,16 @@ def to_numeric(arg, errors="raise", downcast=None):
                     if values.dtype == dtype:
                         break
 
+    if is_numeric_extension_dtype:
```

Comment:
L194 might need to handle the mask

Reply:
Do you mean?
```
            float_32_ind = typecodes.index(float_32_char)
```
I feel like there's a testcase I haven't looked at if yes (the ones I have work as is)